src/Compilers/CSharp/Portable/FlowAnalysis/DataFlowsOutWalker.cs

```diff
@@ -58,6 +58,12 @@ private HashSet<Symbol> Analyze(ref bool badRegion)
             return _dataFlowsOut;
         }
 
+        protected override ImmutableArray<PendingBranch> Scan(ref bool badRegion)
```

Comment:
The analysis was wrong before in the presence of multipass.  But since multipass was rare, we didn't notice.  However, with the way we handle local functions in this PR, we are more likely to have multiple passes and get the wrong answer.  The need for this wqas caught by an existing test.